rig-core/src/providers/anthropic/completion.rs

```diff
@@ -214,7 +214,13 @@ impl completion::CompletionModel for CompletionModel {
 
         if response.status().is_success() {
             match response.json::<ApiResponse<CompletionResponse>>().await? {
-                ApiResponse::Message(completion) => completion.try_into(),
+                ApiResponse::Message(completion) => {
+                    tracing::info!(target: "rig",
+                        "Anthropic completion token usage: {:?}",
```

Comment:
Implement `Diplay` for `Usage` and have it display the stats as a nice string instead of using the debug output which is not very good for logging stuff